web/models/stable_diffusion/stable_args.py

```diff
@@ -145,6 +145,13 @@
 ### Misc. Debug and Optimization flags
 ##############################################################################
 
+p.add_argument(
+    "--use_compiled_scheduler",
+    default=True,
+    action=argparse.BooleanOptionalAction,
+    help="use the default scheduler precompiled into the model if available",
+)
```

Comment:
At the moment I don't think this, nor warmup count has any function. I assume that is to be added once CLI and this is merged.

Reply:
(if that is the plan this comment doesn't need addressing)

Reply:
These flags won't have any effect in the web(there are other flags though like --use_tuned that is useless right now in the web). I have placed them here just to make sure everything matches with CLI, which will make merging process easier.

Reply:
nice nice